site/content/contextual-data-platform/tutorials/evaluate-locally.md

```
@@ -0,0 +1,638 @@
+
+Wait for the Operator pod to become ready and verify it is running:
+
+```sh
+kubectl wait --for=condition=ready pod --selector app.kubernetes.io/name=kube-arangodb-enterprise --namespace arango --timeout=120s
+
+kubectl get pods --namespace arango --selector app.kubernetes.io/name=kube-arangodb-enterprise
+```
+
+Expected output (`x` stands for a varying letter or digit):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)